convert/convert/converters/pdf/pdf_converter.py

```diff
@@ -101,3 +102,53 @@ def _create_neighbor_bbox(self, current_bbox: Bbox) -> Bbox:
         new_x0, new_y0 = x1, y0
         new_x1, new_y1 = (x1 - x0) + x1, y1
         return new_x0, new_y0, new_x1, new_y1
+
+
+class PlainPDFToBadgerdocTokensConverterPytz:
```

Comment:
As I understand it is needed to replase PlainPDFToBadgerdocTokensConverter class by this one